package.json

```diff
@@ -37,6 +37,7 @@
     "fs-extra": "9.0.1",
     "globby": "11.0.1",
     "husky": "4.3.0",
+    "react-lorem-component": "^0.13.0",
```

Comment:
"react-lorem-component": "0",

Reply:
Is not needed IMO since is a external dep

Reply:
remove ^ anyway

Reply:
it might not be in top level dep. Only in the demo needed.